nostalgia/sources/spotify.py

```diff
@@ -31,5 +30,4 @@ def load(cls, file_path="~/nostalgia_data/input/spotify", nrows=None):
 
 if __name__ == "__main__":
     j = Spotify.load()
-    print(j.as_simple())
-    print(j.last_week)
+    print(j.create_sample_data())
```

Comment:
Did you check in & commit the sample data? Did it arrive for you at the right place?

Reply:
Yes, but I'd like to leave creating samples to another branch
I'd like to store these samples somewhere other than src folder

Reply:
Storing them next to the source allows us to package the files in the data and easily find their location. It was a quick solution for now. 

How else to make the connection between a data file and the location of the belonging python class. This was the first solution that came to mind :) 

Maybe to finish this PR including them. Then in a new branch we could reconsider the location of data?

Reply:
After discussed in slack we concluded, that it's better to come up with some project structure for that, appropriate issue will be created